pybadges/__init__.py

```diff
@@ -126,6 +126,7 @@ def badge(
     whole_title: Optional[str] = None,
     left_title: Optional[str] = None,
     right_title: Optional[str] = None,
+    id_suffix: str = '',
```

Comment:
How about `Optional[str] = None` and then we have some code like:

```python
if id_suffix is None:
  id_suffix = str(uuid.uuid4())
```

? 

What do you think? That would mean that the tests need to be updated and the major version of the library bumped?

Or do you think that it is better to leave the default?

Reply:
Thank you for your review!

I think adding a random string seems like a good suggestion.

However, the W3C state that:
> they cannot start with a digit, two hyphens, or a hyphen followed by a digit.
https://www.w3.org/TR/CSS21/syndata.html#characters.

When using `str(uuid.uuid4())`, there are cases where a number exists after `-`.

```py
>>> str(uuid4())
'e489d2a0-39c3-4508-94c5-8337581da1a3'
```

So I think it's better to remove the `-` or leave it as it is.

Also, if keep the current PR, you don't have to update the major version, which is a merit I think.